tests/env-tags.spec.js

```diff
@@ -0,0 +1,75 @@
+const fs = require('fs')
+const { spawnSync } = require('child_process')
+
+describe('env-tags.sh', () => {
+  let branchName, expected
+  afterEach(() => {
+    // Override git command, which is used in `env-tags.sh` to query
+    // the branch name:
+    const bashrcPath = './tests/.bashrc'
+    fs.writeFileSync(bashrcPath, `
+function git() {
+  echo "${branchName}"
+}
+
+export -f git`
```

Comment:
Clever approach to mock the `git` command

Reply:
Thanks! Yeah, getting these tests off the ground was a pain, but I like the result. I hate shell scripting.